tests/test_color_appearance_models.py

```diff
@@ -59,7 +66,9 @@ def load_fixture(file_name):
 
     def check_model_consistency(self, data, output_parameter_dict):
         for data_attr, model_attr in sorted(output_parameter_dict.items()):
-            yield self.check_model_attribute, data.get('Case'), data, model_attr, data[data_attr]
+            yield self.check_model_attribute, data.get("Case"), data, model_attr, data[
+                data_attr
+            ]
```

Comment:
> Notes:
> 
> * Black accepts `--skip-string-normalization` argument, it should reduce diff noise even more.
> * Also black has [psf/black#1054](https://github.com/psf/black/issues/1054), which still contributes a lot of diff noise. They're working on improving of line wrap handling in call args, but still far from ideal.
> * Maybe it makes sense to apply only some of proposed changes and wait until black is out of beta.

About `--skip-string-normalization`: Either you embrace black fully or you do not, the intent for black is to have no option and this particular one came after [very long and painful discussions](https://github.com/psf/black/issues/118). The problem of having such argument is that it will lead to inconsistencies in strings quotation.

> Maybe it makes sense to apply only some of proposed changes and wait until black is out of beta.

You cannot simply cherry-pick some of the changes and dismiss others, then intent is to not talk about code formatting anymore. If the formatting changes because of black changes, so be it. This is the intent.